docs/LogGuide.md

```diff
@@ -923,81 +972,126 @@ Parsed Log Line Examples:
+| 28    | 1000           | target max tp                                                   |
+| 29    | -653.9767      | target x position                                               |
+| 30    | -807.7275      | target y position                                               |
+| 31    | 31.99997       | target z position                                               |
+| 32    | 66480          | caster current hp                                               |
+| 33    | 74095          | caster max hp                                                   |
+| 34    | 4560           | caster current mp                                               |
+| 35    | 4560           | caster max mp                                                   |
+| 36    | 1000           | caster current tp                                               |
+| 37    | 1000           | caster max tp                                                   |
+| 38    | -653.0394      | caster x position                                               |
+| 39    | -807.9677      | caster y position                                               |
+| 40    | 31.99997       | caster z position                                               |
 
 Network ability lines are a combination of raw network data
 (e.g. the `710003` flags and the `9420000` damage)
 and frequently sampled data from memory
 (e.g. the `66480` current hp value and `-653.0394` x position).
 
-This means that there's a number of caveats going on to handling all the data in these lines.  The raw network data is subject to change over time from ff14 servers.  Also, the data from memory may be slightly stale and out of date
+This means that there's a number of caveats going on to handling all the data in these lines.
+The raw network data is subject to change over time from ff14 servers.
+Also, the data from memory may be slightly stale and out of date.
 
-#### Ability Flags
+### Action Effects
 
-Damage bitmasks:
+Each ability may have one or more effects.
+These are indicated by the flagsX and damageX fields, between `targetName` and `targetCurHp`.
+There are eight pairs, each with a 'flags' field and a 'damage' field.
+The damage field is not necessarily always damage.
+For example, for a buff application, it indicates which buff ID is about to be applied.
+
+Damage seems to always be the first field if it is present.
+However, for anything else, it is bad practice to rely on an effect being in a specific position.
+Rather, these should only be treated as an array of pairs, and you should iterate through them to find what you're looking for.
+As an example of why you should not hardcode indices, consider the following.
+
+Here we have a use of Aeolian edge:
+
+`21|2022-09-13T17:25:12.4790000-07:00|10827569|Name Removed|8CF|Aeolian Edge|4000A062|Hegemone|44714003|37120000|A3D|9F8000|53D|9F8000|11B|8CF8000|0|0|0|0|0|0|0|0|rest of line omitted`
```

Comment:
It may help to explain this better if the pairs are indicated in the example line (for both this and the AE+BB example below). Maybe something like this?

```
21|2022-09-13T17:25:12.4790000-07:00|10827569|Name Removed|8CF|Aeolian Edge|4000A062|Hegemone|44714003|37120000|A3D|9F8000|53D|9F8000|11B|8CF8000|0|0|0|0|0|0|0|0|rest of line omitted
                                                                                             |1st Pair         |2nd Pair  |3rd Pair  |4th Pair   |5P |6P |7P |8P |
```

(I couldn't make this an actual GH suggestion because GH didn't like nesting a code block in a suggestion block)

Reply:
Done